JoomGalleryfriends/JG4-dev
# Remove ACL for images

This PR fixes the problem reported in issue #252.

With this PR applied, there is no Permissions tab anymore in the image form view. Instead there are two tabs in the category form view. One for permissions regarding the category itself and one regarding all images within this category.

![grafik](https://github.com/user-attachments/assets/8d3fbfbc-6f3e-42f4-9316-d1ca31a62752)


# Testing instructions
**You need to install a new, fresh JoomGallery in order for this PR to work!**

Instead of having permissions for each image, we have now permissions for all images within one category.
Create different users and apply different usergroups to them. Log in with this user and check if the permissions are applied correctly according to the permissions you set in the configuration for this usergroup.

**Attention!**
Forntend forms may not work correctly and throw errors. This is most probably due to the fact that the forms itself are not yet correctly set up (missing form fields, wrong URLs, wrong variables) in their corresponding template. Fixing these templates will not be part of this PR.

Following permissions should be tested at least:

## Category
- Create / Own
- Delete / Own
- Edit / Own
- Edit State

## Image
- Upload / Own
- Delete / Own
- Edit / Own
- Edit State
